packages/bits/src/lib/date-picker/date-picker-inner.component.html

```diff
@@ -1,3 +1,3 @@
-<div *ngIf="datepickerMode" class="nui-datepicker-inner" role="application">
+<div *ngIf="datepickerMode" class="nui-datepicker-inner" role="dialog">
```

Comment:
I don't think `dialog` can be used here because date-picker-inner can either be inline or inside an overlay. Maybe date-picker-inner can have an input to specify the correct role?

https://github.com/solarwinds/nova/blob/main/packages/bits/src/lib/date-picker/date-picker.component.html#L28

Reply:
You are right! will add, thanks